docs/windows.md

```diff
@@ -426,6 +368,130 @@ antrea-agent-windows-6hvkw                             1/1     Running     0
 kube-proxy-windows-2d45w                               1/1     Running     0          102s
 ```
 
+### Add Windows kube-proxy DaemonSet (only for Kubernetes versions prior to 1.26)
+
+Starting from Kubernetes 1.26, Antrea will no longer support Windows kube-proxy
+because kube-proxy userspace mode is deprecated and kernelspace mode does not
+work with Antrea. You should skip this step on the kubernetes cluster starting
+from 1.26 and enable Antrea proxy `proxyAll` mode for Windows agent.
+
+Add Windows-compatible versions of kube-proxy by applying `kube-proxy.yaml`.
+Download `kube-proxy.yaml` from the Kubernetes official repository and set
+the kube-proxy version.
+
+An example command that downloads and applies `kube-proxy.yml` is like below:
```

Comment:
Ditto

Reply:
The following command deploys kube-proxy v1.20.0?

Reply:
Yes, it's the latest compatible kubeproxy image we can use.